modules/Microsoft.Network/privateLinkServices/.test/parameters.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-az-pls-001"
+        },
+        "lock": {
+            "value": "CanNotDelete"
+        },
+        "ipConfigurations": {
+            "value": [
+                {
+                    "name": "pls01",
+                    "properties": {
+                        "primary": true,
+                        "privateIPAllocationMethod": "Dynamic",
+                        "subnet": {
+                            "id": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/virtualNetworks/adp-<<namePrefix>>-az-vnet-x-001/subnets/<<namePrefix>>-az-subnet-x-001"
+                        }
+                    }
+                }
+            ]
+        },
+        "loadBalancerFrontendIpConfigurations": {
+            "value": [
+                {
+                    "id": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/loadBalancers/adp-<<namePrefix>>-az-lb-internal-001/frontendIPConfigurations/privateIPConfig2"
+                }
+            ]
+        },
+        // "extendedLocation": {
```

Comment:
Is this commented section intentional?

Reply:
Yes, `extendedLocation` capability is still in private preview, however, given that its `name` key accepts specific IDs only, I thought having this object commented will be a good reference.
However, let me know if this code sample should also move to readme.

Reply:
I just checked the APIs for PLS, this property has been there since `2020-11-01`. I would suggest we continue to test it and provide the information about the feature in the documentation parameter usage section. Is there some documentation for it to include as well?

Reply:
Yes, tested it, it is failing no matter which parent region we choose.
Yes, the parameter usage section has hyperlink to this MS doc - [Azure Edge Zone ID](https://docs.microsoft.com/en-us/azure/public-multi-access-edge-compute-mec/key-concepts#azure-edge-zone-id)

Note that right now I have moved this from the `parameters.json` file. Let me know for any changes.

Reply:
Ok now I have a much better understanding of this. Thanks for sharing @kavishshivhare . Please feel free to resolve